ansible/roles/software/ckan/catalog/ckan-app/tasks/web.yml

```
@@ -4,6 +4,25 @@
   notify: reload apache2
   when: not ckan_uses_gunicorn
 
+- name: Copy CKAN server_start.sh
+  copy:
+    src: "{{ app_source_path }}/ckan/setup/server_start.sh"
+    dest: /etc/ckan/server_start.sh
+    mode: preserve
```

Explicitly set the mode.

```suggestion
    mode: 755
```